exonum/src/helpers/fabric/details.rs

```diff
@@ -420,18 +456,16 @@ impl GenerateNodeConfig {
         (external_address, listen_address)
     }
 
-    fn get_passphrase(context: &Context, secret_key_type: SecretKeyType) -> ZeroizeOnDrop<Vec<u8>> {
-        let key = match secret_key_type {
-            SecretKeyType::Consensus => EXONUM_CONSENSUS_PASS,
-            SecretKeyType::Service => EXONUM_SERVICE_PASS,
-        };
-        context
-            .get_flag_occurrences(NO_PASSWORD)
-            .map(|_| ZeroizeOnDrop("".to_string()))
-            .or_else(|| env::var(key).map(ZeroizeOnDrop).ok())
-            .or_else(|| prompt_passphrase(secret_key_type).ok())
-            .map(|s| ZeroizeOnDrop(s.0.as_bytes().to_vec()))
-            .unwrap()
+    fn get_passphrase(
+        context: &Context,
+        method: PassInputMethod,
+        secret_key_type: SecretKeyType,
+    ) -> ZeroizeOnDrop<String> {
+        if context.get_flag_occurrences(NO_PASSWORD).is_some() {
+            ZeroizeOnDrop("".to_owned())
```

Comment:
Can you explain in comment what the following line do?

Reply:
Returns empty password.